fuel/bin/fuel_convert.py

```diff
@@ -58,12 +58,17 @@ def main(args=None):
         subparser.add_argument(
             "-o", "--output-directory", help="where to save the dataset",
             type=str, default=os.getcwd(), action=CheckDirectoryAction)
+        subparser.add_argument(
+            "-r", "--output_filename", help="new name of the created dataset",
+            type=str, default=None)
         # Allows the parser to know which subparser was called.
         subparser.set_defaults(which_=name)
         convert_functions[name] = fill_subparser(subparser)
 
     args = parser.parse_args(args)
     args_dict = vars(args)
+    if args_dict['output_filename'] is not None:
```

Comment:
Maybe add something like `and os.path.splitext(args_dict['output_filename'])[1] not in ('.hdf5', '.hdf', '.h5')`?